### ISSUE TITLE
gRPC error: "Request error :cannot connect (408 Timeout): the gRPC port may not be correctly opened in the server"

### ISSUE THREAD

**Issue by wallacerob** [pydio/cells-sync#15]
* Cells Sync 0.9.0 on macOS 10.15
* Cells Home 2.0.0-rc2 on remote server (Ubuntu 18.04.3)
* Docker image pydio/cells:2.0.0-rc2 mapped to host port 8080
* nginx proxy passing https to localhost:8080

```
2019-11-01T21:56:48.513-0500	ERROR	http-server	Request error :context deadline exceeded
2019-11-01T21:56:51.897-0500	ERROR	http-server	Request error :cannot connect (408 Timeout): the gRPC port may not be correctly opened in the server
```
__________________________
I can authenticate with my Cells server and add account to Sync. When I try to create a task I get the error when attempting to browse the remote tree. 

Server log message:
```
Ts : 1572665725
Level : error
Logger : pydio.grpc.tasks
Msg : Streamer PutTaskStream - {"id":"go.micro.client","code":500,"detail":"none available","status":"Internal Server Error"}
SpanUuid : span-2
```
_________________________

edit: OSes

**Comment by Vallefor** [comment]
This is exacly my issue from here: https://github.com/pydio/cells-sync/issues/11
Try to follow this poroxy settings for nginx: https://github.com/pydio/cells-sync/files/3784576/nginx_reverse_proxy.pdf (this doc file not for docker but it is possible to understand what to do)
The highlight is that you need to proxy pass grpc port too and cells should running with ssl option (with self signed cert if you want).

**Comment by wallacerob** [comment]
@Vallefor Thanks very much for this information. I now have sync working with sync0.9.1 & cells2.0.0. 

It was indeed a proxy error for the grpc port, with further complication using docker. I reinstalled cells with the 2.0 release, and am using the precompiled sync binary from Pydio. 

After mapping the external grpc container port to host, and proxying from nginx to that host port, I was able to get sync working.
